packages/loopring_v3/contracts/impl/libexchange/ExchangeData.sol

```diff
@@ -188,6 +189,38 @@ library ExchangeData
     function MIN_AGE_PROTOCOL_FEES_UNTIL_UPDATED() internal pure returns (uint32) { return 1 days; }
     function GAS_LIMIT_SEND_TOKENS() internal pure returns (uint32) { return 30000; }
 
+    function getConstants()
+        public
+        pure
+        returns(
+            bytes32 merkleRoot,
+            uint[19] memory constants
+        )
+    {
+        merkleRoot = GENESIS_MERKLE_ROOT();
```

Comment:
I don't think there's any need to special case `merkleRoot` here, putting it in the constants array should be fine.

Reply:
done